subprojects/architecture-test/src/test/java/org/gradle/architecture/library/freeze/LineNumberIgnoringViolationStore.java

```diff
@@ -36,7 +36,7 @@ public class LineNumberIgnoringViolationStore implements ViolationStore {
     public LineNumberIgnoringViolationStore() {
         // We can instantiate TextFileBasedViolationStore directly once ArchUnit 1.0.2 is released,
         // since TextFileBasedViolationStore became public type in https://github.com/TNG/ArchUnit/pull/1046.
```

Comment:
Remove comment since no longer relevant?

Reply:
Of course ... thanks!